examples/console/main.py

```diff
@@ -9,34 +28,43 @@
 
 import tornado.web
 
-# Install the pyzmq ioloop. This has to be done before anything else from
-# tornado is imported.
+# Install the pyzmq ioloop. Must be done after importing tornado.web and
+# before importing any additional tornado modules
 from zmq.eventloop import ioloop
 ioloop.install()
 
 PORT = 8765
+"""int: Port number of web application"""
 
 
 class MainPageHandler(tornado.web.RequestHandler):
+    """Handle requests between the main app page and notebook server."""
 
     def initialize(self, base_url):
+        """Intitialize the base URL of the handler."""
         self.base_url = base_url
 
     def get(self):
+        """Get the main page for the application's interface."""
         return self.render("index.html", static=self.static_url,
                            base_url=self.base_url)
 
 
 def main(argv):
+    """Start the 'console' plugin.
```

Comment:
That's reasonable yet likely confusing to the end user (new developer). For example, a sprint participant will need to grok ``plugin``, ``Application``, ``extensions``, ``service providers``, etc.

To help with the understanding, we should specify exactly which "files comprise the plugin" (are they only the ones in ``src``?) and then indicate the rest of the files are simply needed to run the "stand alone example".